CHANGELOG.md

```diff
@@ -1,11 +1,18 @@
 Note: This is in reverse chronological order, so newer entries are added to the top.
 
+Swift 5.8
+-----------
+
+* [#5728]
+
+  In packages that specify resources using tools version 5.8 or later, the generated resource bundle accessor will import `Foundation.Bundle` for its own implementation only. _Clients_ of such packages therefore no longer silently import `Foundation`, preventing inadvertent use of Foundation extensions to standard library APIs that lead to unexpected code size increases.
```

Comment:
I think I would cut the `preventing` part? Unless I am misunderstanding something, this does not solve the code size increase, since Foundation will still be a link-time dependency of any end product since we are not changing anything about the generated code which does use Foundation. Or is there some kind of optimization we're enabling here?

Reply:
The initial forum post identified the bug on the basis of an unexpected code size increase in a WASM binary. Bisecting their commit history identified the unintentional use of Foundation extensions to standard library types due to this implicit import as the reason for the code size increase, and removing use of those APIs fixed the problem.

Reply:
maybe "help avoiding"

Reply:
It doesn't just "help"--the change in question actually eliminates, disables, renders impossible the use of such Foundation extensions without explicitly importing `Foundation`. That is in fact the salient thing that users probably want to know about this change; all of the preceding text is arguably less relevant implementation detail.

Reply:
How about this: `preventing inadvertent use of Foundation extensions to standard library APIs, which helps to avoid unexpected code size increases`.

Reply:
Love it!